dashboard/client/requests_over_time/requests_over_time.js

```diff
@@ -0,0 +1,125 @@
+import d3 from 'd3';
+import nvd3 from 'nvd3';
+import _ from 'lodash';
+
+Template.requestsOverTime.onRendered(function () {
+  // Get reference to template instance
+  const templateInstance = Template.instance();
+
+  templateInstance.elasticsearchData = new ReactiveVar();
+  templateInstance.chartData = new ReactiveVar();
+
+  const queryParams = {
+    size: 0,
+    body: {
+      query: {
+        filtered: {
+          query: {
+            bool: {
+              should: [
+                {
+                  wildcard: {
+                    request_path: {
+                      // Add '*' to partially match the url
+                      value: '/*',
+                    },
+                  },
+                },
+              ],
+            },
+          },
+        },
+      },
+      aggs: {
+        requests_over_time: {
+          date_histogram: {
+            field: 'request_at',
+            interval: 'month',
+            format: 'yyyy-MM-dd'
+          },
+        },
+      },
+    },
+  };
+
+  // Initialize chart
+  const chart = nvd3.models.historicalBarChart();
+
+  // configure chart
+  chart
+    .xScale(d3.time.scale())
+    .margin({left: 100, bottom: 100})
+    .showXAxis(true);
+
+  // configure x-axis settings for chart
+  chart.xAxis
+    .axisLabel('Days')
+    .tickFormat(function (d) {
+      // Format tickmarks based on timescale
+      return d3.time.format('%x')(new Date(d))
```

Comment:
This code does not need to inherit `this` from the parent context, so an arrow function is not necessary.